src/main/java/com/bravo/user/service/PaymentService.java

```diff
@@ -0,0 +1,31 @@
+package com.bravo.user.service;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.model.dto.PaymentDto;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PaymentService {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PaymentService.class);
+
+  private final PaymentRepository paymentRepository;
+  private final ResourceMapper resourceMapper;
+
+  public PaymentService(PaymentRepository paymentRepository, ResourceMapper resourceMapper) {
+    this.paymentRepository = paymentRepository;
+    this.resourceMapper = resourceMapper;
+  }
+
+  public List<PaymentDto> retrieveByUserId(final String userId) {
```

Comment:
This method should accept a Pageable parameter

Reply:
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/29028698/210009261-1ce00e4f-c6ab-43d7-9bcf-c229109f5064.png">
<img width="888" alt="Screen Shot 2022-12-29 at 12 43 20 PM" src="https://user-images.githubusercontent.com/29028698/210009291-9a1b6761-f9c3-4998-ba47-b8ba8836d5fe.png">
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/29028698/210009321-1e4b1b6d-e301-4620-92a4-37758daff2c2.png">